templates/static/en/about.html

```diff
@@ -19,13 +19,11 @@
 {% block content %}
 <div class="superAbout">
     {% if not request.user_agent.is_mobile %}
-        {% include '_sidebar.html' with whichPage='about' title="What is Sefaria?" heTitle="מהי ספריא" %}
+        {% include '_sidebar.html' with whichPage='about' %}
```

Comment:
Note that on this page the `<header>` (line 26) only has an English translation, not a Hebrew one. Can you confirm that it does/does not render the Hebrew header please?

Reply:
It doesn't render in Hebrew.  When you switch to a Hebrew interface language, /he/about.html loads instead.